pkg/aws/services/rgt.go

```diff
@@ -17,6 +27,42 @@ func NewRGT(session *session.Session) RGT {
 	}
 }
 
+var _ RGT = (*defaultRGT)(nil)
+
 type defaultRGT struct {
 	resourcegroupstaggingapiiface.ResourceGroupsTaggingAPIAPI
 }
+
+func (c *defaultRGT) GetResourcesAsList(ctx context.Context, input *resourcegroupstaggingapi.GetResourcesInput) ([]*resourcegroupstaggingapi.ResourceTagMapping, error) {
+	var result []*resourcegroupstaggingapi.ResourceTagMapping
+	if err := c.GetResourcesPagesWithContext(ctx, input, func(output *resourcegroupstaggingapi.GetResourcesOutput, _ bool) bool {
+		for _, i := range output.ResourceTagMappingList {
+			result = append(result, i)
+		}
+		return true
+	}); err != nil {
+		return nil, err
+	}
+	return result, nil
+}
+
+func ParseRGTTags(tags []*resourcegroupstaggingapi.Tag) map[string]string {
+	result := make(map[string]string, len(tags))
+	for _, tag := range tags {
+		result[aws.StringValue(tag.Key)] = aws.StringValue(tag.Value)
+	}
+	return result
+}
+
+func NewRGTTagFilters(tags []tracking.TagFilter) []*resourcegroupstaggingapi.TagFilter {
```

Comment:
this `NewRGTTagFilters ` function should be put into tracking package. the `pkg/aws/service` should be free of our business logic, so it don't know "tracking" package at all.